external-api/src/helpers/types/estore.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Represents a category in the eStore.
+ * Possible values are 'sites', 'buyer', 'deals', 'facilities', and 'documents'.
+ */
+export type Category = 'sites' | 'buyer' | 'deals' | 'facilities' | 'documents';
```

Comment:
Could we extract this so we have 

```suggestion
export const CATEGORIES = {
  SITE: 'sites',
  BUYER: 'buyer',
  DEAL: 'deals',
  FACILITY: 'facilities',
  DOCUMENT: 'documents',
} as const;
```
(the as const provides stronger typing)

We can then do
```
export type CategoryType = ValuesOf<typeof CATEGORIES>;
```

to get an auto updating type for it


Also -- I'm a little confused about how this relates to `ENDPOINT` enum and if it does? It seems to be missing `TERM: 'terms/facilities'`. If they are linked -- I'd be tempted to extend from the smaller type, or omit from the larger type (which ever fits business logic more)?

ie 

```suggestion
export const CATEGORIES_FOR_FOO = {
  SITE: 'sites',
  BUYER: 'buyer',
  DEAL: 'deals',
  FACILITY: 'facilities',
  DOCUMENT: 'documents',
} as const;


const CATEGORIES_FOR_BAR = {
    TERM: 'terms/facilities
} as const

export const CATEGORIES = {
    ...CATEGORIES_FOR_FOO,
    ...CATEGORIES_FOR_BAR
} as const

export type CategoriesForFooType = ValuesOf<typeof CATEGORIES_FOR_FOO>;
type CategoriesForBaaType = ValuesOf<typeof CATEGORIES_FOR_BAR>;
export type CategoriesType = CategoriesForFooType | CategoriesForBaaType;

```

Reply:
Same as above and good spot have added `term/facilities`.